SonarLint.VisualStudio.Integration.sln

```diff
@@ -26,6 +26,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		src\AssemblyInfo.Shared.cs = src\AssemblyInfo.Shared.cs
 		Directory.Build.props = Directory.Build.props
 		Directory.Build.targets = Directory.Build.targets
+		nuget.config = nuget.config
```

Comment:
Just added it to the solution, I think it makes more sense to see it directly when there is a build failure due to new unsigned package.